app/views/admin/configs/show.html.erb

```diff
@@ -36,41 +36,42 @@
         <%= form_for(SiteConfig.new, url: admin_config_path) do |f| %>
           <fieldset class="grid gap-4">
 
-          <% VerifySetupCompleted::MANDATORY_CONFIGS.each do |config_key| %>
+          <% VerifySetupCompleted::MANDATORY_CONFIGS.each do |config_key, settings_model| %>
             <%# we need to list the config as separate fields if the data structure is a Hash %>
-            <% if SiteConfig.public_send(config_key).is_a?(Hash) %>
+            <% placeholder_module = Constants.const_get(settings_model.name) %>
+            <% if settings_model.public_send(config_key).is_a?(Hash) %>
               <%= admin_config_label config_key.to_s %>
               <div class="form-group">
                 <%= f.fields_for config_key do |a_field| %>
-                  <% SiteConfig.public_send(config_key).each do |key, value| %>
+                  <% settings_model.public_send(config_key).each do |key, value| %>
```

Comment:
This whole block of changes is due to the related change in `VerifySetupCompleted`.